operators/swiss-staking/metadata.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "Swiss Staking AG",
   "website": "https://swiss-staking.ch",
-  "description": "Swiss Staking has over ~100,000 delegators across 20+ networks and runs its Eigenlayer and Ethereum operator in a former military bunker in Switzerland for maximal security and reliability.",
+  "description": "Swiss Staking has over 100,000 delegators across 20 networks and runs its Eigenlayer and Ethereum operator in a former military bunker in Switzerland for maximal security and reliability.",
```

Comment:
I think this will work in latest release - https://github.com/Layr-Labs/eigenlayer-cli/releases/tag/v0.7.2 - try it out. you don't need to remove this now :)

Reply:
@creeea

Reply:
doesn't work :)
I think char ~ is the issue

Reply:
@shrimalmadhur

Reply:
works